src/Tribe/Events/Attendees_List.php

```diff
@@ -313,4 +313,23 @@ protected function is_showing_attendee_list_with_classical_editor( WP_Post $post
 
 		return $is_visible_by_meta || $has_attendee_list_shortcode;
 	}
+
+	/**
+	 * Get total number of attendees that are confirmed.
+	 *
+	 * @since TBD
+	 *
+	 * @param int|string $post_id The post or event that is checked.
+	 *
+	 * @return int Total number of attendees attending the event.
+	 */
+	public function get_attendance_counts( $post_id ) {
+		$attendees_orm = tribe_attendees();
+
+		$attendees_orm->per_page( -1 )
+		              ->where( 'event', $post_id )
+		              ->where( 'rsvp_status__or_none', 'yes' );
+
+		return $attendees_orm->count();
```

Comment:
If we use found(), it should do the same thing without having to fetch ALL attendees. We don't want to fetch all attendees and get the total count, we just want to know how many are in the database.

```suggestion
		return $attendees_orm->where( 'event', $post_id )
		              ->where( 'rsvp_status__or_none', 'yes' )
		              ->found();
```